clusterloader2/pkg/imagepreload/manifests/daemonset.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       initContainers:
       {{range $idx, $image := .Images}}
       - name: preload-{{$idx}}
-        image: gcr.io/k8s-testimages/perf-tests-util/containerd
+        image: gcr.io/k8s-testimages/perf-tests-util/containerd:latest
```

Comment:
Let's don't use latest - this means that it can change behind the scenes and change behavior of our tests without us knowing anything potentially.

Please use a sepecific tag (IIRC Matt pushed 0.1)